### ISSUE TITLE
Fix pydoctor epydoc errors in twisted.positioning

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8552]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#8552|
|Type|enhancement|
|Created|2016-07-03 14:51:22Z|
|Branch|https://github.com/twisted/twisted/tree/8552-pydoctor-positioning|

<details><summary>Searchable metadata</summary>

```
trac-id__8552 8552
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__None None
branch__8552_pydoctor_positioning 8552-pydoctor-positioning
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467557482544190 1467557482544190
changetime__1467701068225190 1467701068225190
version__None None
owner__adiroiban adiroiban
cc__lvh
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban removed owner|
|-|-|

1. ipositioning.IPositioningSentenceProducer don't exist and the existing test docstring was very brief... it like it is just a stub so I have removed it.

--------

Check that 

```
./bin/admin/build-apidocs . apidocs/ | grep positioning

```

produces no output


you can also  check the api-documentation stdio output in the builder

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc set owner to @adiroiban|
|-|-|

OK

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset a6180f43b356bc98cacedd59f5d532c4b91e60c7
```
#!CommitTicketReference repository="" revision="a6180f43b356bc98cacedd59f5d532c4b91e60c7"
Merge 8552-pydoctor-positioning: Fix pydoctor errors in twisted.positioning.

Author: adiroiban
Reviewer: rodrigc
Fixes: #8552
```
